src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -308,6 +311,11 @@ void ov::npuw::LLMCompiledModel::set_property(const ov::AnyMap& properties) {
 
 ov::Any ov::npuw::LLMCompiledModel::get_property(const std::string& name) const {
     OPENVINO_SUPPRESS_DEPRECATED_START
+    if (name == ov::intel_npu::npuw::llm::prefill_config.name() ||
```

Comment:
I don't believe it's really needed, see comment above

Reply:
`get_property()` might be not needed at all here, so as it is a redudant functionality, I suppose to at least handle everything in a unified way here to not create a mess.

Reply:
Keys provided to `LLM` pipeline must not be `properties`, so there won't be any mess

Reply:
And what they should be in this situation? And how they should be passed into `core.compile_model()` then?

Reply:
`compile_model` accept `ov::AnyMap`, doesn't it? https://docs.openvino.ai/2024/api/c_cpp_api/classov_1_1_core.html

GenAI pipeline does it exactly this way